vernac/attributes.ml

```diff
@@ -293,6 +293,14 @@ let deprecation_parser : Deprecation.t key_parser = fun ?loc orig args ->
 
 let deprecation = attribute_of_list ["deprecated",deprecation_parser]
 
+let library_info_parser : string key_parser = fun ?loc orig args ->
+  assert_once ?loc ~name:"information" orig;
+  match args with
+  | VernacFlagLeaf (FlagString str) -> str
+  |  _ -> CErrors.user_err ?loc (Pp.str "Ill formed “information” attribute")
+
+let library_info = attribute_of_list ["information", library_info_parser]
+
```

Comment:
Belongs to https://github.com/coq/coq/pull/18248 (once rebased on top of the current PR).

Reply:
I don't understand your remark. It is needed for the current PR.

Reply:
C.f. https://github.com/coq/coq/pull/18193#discussion_r1395854202